pkg/archive/archive_test.go

```diff
@@ -67,7 +67,14 @@ func TestIsArchivePathDir(t *testing.T) {
 }
 
 func TestIsArchivePathInvalidFile(t *testing.T) {
-	cmd := exec.Command("sh", "-c", "dd if=/dev/zero bs=1K count=1 of=/tmp/archive && gzip --stdout /tmp/archive > /tmp/archive.gz")
+	var bsVal string
+	if runtime.GOOS == "solaris" {
+		bsVal = "bs=1"
```

Comment:
Are you sure? The man page says that would be a 1 byte blocksize! `bs=1024` should work for all platforms.